hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/upload/UploadConfirmationWindow.java

```diff
@@ -619,19 +622,23 @@ private void createArtifact(final String itemId, final String filePath, final Ar
             final SoftwareModule baseSw) {
 
         final File newFile = new File(filePath);
-        final Item item = tabelContainer.getItem(itemId);
-        final String sha1Checksum = ((TextField) item.getItemProperty(SHA1_CHECKSUM).getValue()).getValue();
-        final String md5Checksum = ((TextField) item.getItemProperty(MD5_CHECKSUM).getValue()).getValue();
+        final Item item = tableContainer.getItem(itemId);
+        // We have to make sure that null is assigned to sha1Checksum and
+        // md5Checksum if no alphanumeric value is provided. Empty String will
+        // fail
+        final String sha1Checksum = org.apache.commons.lang3.StringUtils
```

Comment:
Why here is `org.apache.commons.lang3.StringUtils` but in line e.g. 193 https://github.com/eclipse/hawkbit/pull/633/files#diff-ab32214bd37aa42bd0e5b1f92a91a1eeR193 is `org.springframework.util.StringUtils` used. I would use spring overall.